.env.example

```
@@ -7,20 +7,25 @@ APP_PORT = "REPLACE_WITH_APP_PORT"
 APP_ENV = "REPLACE_WITH_APP_ENV"
 ALLOWED_ORIGINS = "REPLACE_WITH_ALLOWED_ORIGINS"
 RUST_LOG = "REPLACE_WITH_RUST_LOG"
+DOWNLOAD_DIR = "REPLACE_WITH_DOWNLOAD_DIR"
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)